src/frontends/tensorflow/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@
 
 # TODO: Add LINKABLE_FRONTEND option when tensorflow frontend directory is moved to openvino folder
 ov_add_frontend(NAME tensorflow
+                SHUTDOWN_PROTOBUF
```

Comment:
do we need this as an option or we can detect this automatically in `ov_add_frontend`? We know, that we have proto files in current directly and can take this into account

Reply:
We can detect automatically by 'if (proto_files) in cmake/developer_package/frontends/frontends.cmake for tensorflow and paddle. But onnx is different, onnx link the protobuf in src/frontends/onnx/onnx_common/CMakeLists.txt. In order to detect the protobuf in onnx, the better way I came up with is to check whether the TARGET_NAME is 'onnx' or check whether link the library 'onnx_common'. But this behavior make 'onnx' an specific target to to add the shutdown files, if so I think it's better to add an option to let target choose whether to add the shutdown files. @ilya-lavrenov what do you think?

Reply:
agree, forgot about ONNX differences